RELEASE.md

```diff
@@ -0,0 +1,3 @@
+release type: patch
+
+This release fixes generic input type convertsion so that generic type can be used as type of arguments.
```

Comment:
`````suggestion
This release fixes support for generic types so that now we can also use generics for input types:

```python
T = typing.TypeVar("T")

@strawberry.input
class Input(typing.Generic[T]):
    field: T

@strawberry.type
class Query:
    @strawberry.field
    def field(self, input: Input[str]) -> str:
        return input.field
```
`````